### DISCUSSION TITLE
NoiseModelling 4.0.5

### DISCUSSION THREAD

**Discussion by nicolas-f** [Universite-Gustave-Eiffel/NoiseModelling#608] [General]
## What's Changed
* fix a rare osm import node NullPointer error by @Nitnelav in https://github.com/Universite-Gustave-Eiffel/NoiseModelling/pull/548
* Update Import_Symuvia.groovy for ANR SYMEXPO project by @pierromond in https://github.com/Universite-Gustave-Eiffel/NoiseModelling/pull/549
* Fix building grid 3D with pop column, add unit test by @nicolas-f in https://github.com/Universite-Gustave-Eiffel/NoiseModelling/pull/550
* Fix buildings receivers by @nicolas-f in https://github.com/Universite-Gustave-Eiffel/NoiseModelling/pull/553
* fix plotting of directivity by @nicolas-f in https://github.com/Universite-Gustave-Eiffel/NoiseModelling/pull/557
* TRIANGLES in Regular Grid by @nicolas-f in https://github.com/Universite-Gustave-Eiffel/NoiseModelling/pull/563
* small fix & improvements by @Nitnelav in https://github.com/Universite-Gustave-Eiffel/NoiseModelling/pull/567
* More explicit error message by @nicolas-f in https://github.com/Universite-Gustave-Eiffel/NoiseModelling/pull/573
* Add input param by @gpetit in https://github.com/Universite-Gustave-Eiffel/NoiseModelling/pull/571
* Bump jackson-databind from 2.9.10.7 to 2.12.7.1 in /noisemodelling-propagation by @dependabot in https://github.com/Universite-Gustave-Eiffel/NoiseModelling/pull/566
* Update Noise_Map_From_Point_Source.rst by @mrerdem in https://github.com/Universite-Gustave-Eiffel/NoiseModelling/pull/581
* Update nm_terminal.bash by @pierromond in https://github.com/Universite-Gustave-Eiffel/NoiseModelling/pull/587
* Fix kml path by @nicolas-f in https://github.com/Universite-Gustave-Eiffel/NoiseModelling/pull/590
* Thanks to GeoClimate, much more Buildings are kept...  by @pierromond in https://github.com/Universite-Gustave-Eiffel/NoiseModelling/pull/578
* Repair major diffraction issues (horizontal through berms or topograp… by @pierromond in https://github.com/Universite-Gustave-Eiffel/NoiseModelling/pull/595
* Refactor emission module by @pierromond in https://github.com/Universite-Gustave-Eiffel/NoiseModelling/pull/596
* Update Input_receivers.rst by @nicolas-f in https://github.com/Universite-Gustave-Eiffel/NoiseModelling/pull/600
* avoid NPE if no description in wps, about #597 by @nicolas-f in https://github.com/Universite-Gustave-Eiffel/NoiseModelling/pull/599
* fix ComputeCnossosRays.java by @pierromond in https://github.com/Universite-Gustave-Eiffel/NoiseModelling/pull/603

## New Contributors
* @mrerdem made their first contribution in https://github.com/Universite-Gustave-Eiffel/NoiseModelling/pull/581

**Full Changelog**: https://github.com/Universite-Gustave-Eiffel/NoiseModelling/compare/v4.0.3...v4.0.5

<hr /><em>This discussion was created from the release <a href='https://github.com/Universite-Gustave-Eiffel/NoiseModelling/releases/tag/v4.0.5'>NoiseModelling 4.0.5</a>.</em>

**Comment by Nitnelav** [comment]
Here is the Windows installer : https://github.com/Nitnelav/NoiseModelling-installer/releases/tag/v4.0.5
